### ISSUE TITLE
Post language selection

### ISSUE THREAD

**Issue by osmaa** [elk-zone/elk#876]
I'm a bilingual user who prefers to have their software in English, but often posts in Finnish. For me:

* Elk always defaults to expect my post language is in English, presumably because that's my interface language(?), regardless of the language of any current discussion
* Finnish is nowhere near my language selection dropdown top, and I can't find a way to get it there, though my home instance's native web UI places it as the second choice
* The language pulldown menu icon gives me no hint on what the current language choice is, forcing me to open it to see

Suggested change:

* Mastodon has my "frequently used languages" in the UI, though I can't place it in the API.. If possible, follow the same https://github.com/mastodon/mastodon/search?q=frequentlyusedlanguages&type=code
* show the currently selected language - at least for users who have more than one language indicated
* for a Reply, Mastodon itself defaults to the thread's language, unless the user has translated it: https://github.com/mastodon/mastodon/commit/f70bdba9264bd7c572cee3c45421733919b7d03c

**Comment by moeffju** [comment]
To add to this beyond my 👍🏼, I myself as well as many people I know frequently post in multiple languages. The current language selector in most Mastodon UIs is really hard to use in these cases; at best, they pin English close to the top so I can choose my primary language as my default post language and have that and English at the top of the list for quick switching. A third language breaks this flow.

What I've suggested to Tusky and the official Mastodon UI is that users can choose any number of languages to pin to the top of their list. For the official Mastodon web UI, the content languages setting could be repurposed, but having a separate setting is likely better for flexibility.

One other thing I would like to propose for Elk specifically would be a way to change language quickly by in-band signaling, e.g. by typing a specific sequence of characters, or at least having a hotkey for the language selector (there might be one already, but I found no documentation and there is no mention in the tool tip). It's important for me personally to tag my posts with the correct language, but it's a bit cumbersome on all clients I currently use.
